src/tribol/mesh/MfemData.hpp

```diff
@@ -694,7 +703,7 @@ class MfemMeshData
    */
   integer GetMesh1NE() const
```

Comment:
Strongly consider checking to see if the data is there and in fact updated.

Reply:
Fixed.  The `GetUpdateData()` method does this check.